### ISSUE TITLE
Differing formulas for CU calculations in wiki versus ThreeBot

### ISSUE THREAD

**Issue by scottyeager** [threefoldfoundation/info_foundation#225]
I am actually not sure if this issue is in the wiki or ThreeBot's code, but each has a different way for calculating CUs.

In the table on the [CU wiki page](https://wiki.threefold.io/#/cloud_units_4), we see:
![image](https://user-images.githubusercontent.com/17682591/88355397-4264cc00-cd19-11ea-9674-62fab2c1d1c4.png)

In ThreeBot’s [code](https://github.com/threefoldtech/jumpscaleX_threebot/blob/6121737a21f9af5549815c7fbfb60e3f200082a3/ThreeBotPackages/threebot/farmmanagement/frontend/components/nodeinfo/nodeinfo.js), we see:
```
cu : Math.round(Math.min(r.mru/4*0.95, r.cru * 2) * 1000) / 1000,
su : Math.round(( r.hru / 1093 + r.sru / 91) * 1000) / 1000,
```

The wiki page on Cloud Units can also be improved, regardless of it's correctness. CU formulas are very small, down in the corner, and there is no mention of how passmark ratings relate to CUs.

**Comment by despiegk** [comment]
thank you to find this, amazing, will check on Monday which one is the right one

**Comment by scottyeager** [comment]
Absolutely—glad I could help here. The wiki page is looking much better!

**Comment by sasha-astiadi** [comment]
thank you @scottyeager , we're trying hard to be better organised :)
